jaeger/cmd/uninstall.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/linkerd/linkerd2/pkg/k8s/resource"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	yamlSep = "---\n"
+)
+
+func newCmdUninstall() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "uninstall",
+		Args:  cobra.NoArgs,
+		Short: "Output Kubernetes resources to uninstall the Linkerd-jaeger extenion",
+		Long: `Output Kubernetes resources to uninstall the Linkerd-jaeger extenion.
+
+This command provides all Kubernetes namespace-scoped and cluster-scoped resources (e.g services, deployments, RBACs, etc.) necessary to uninstall the Linkerd-jaeger extenion.`,
```

Comment:
I don't see namespace-scoped resources in the `uninstall` output. I only see the cluster-scoped resources and the `linkerd-jaeger` namespace resource. None of the services or deployments are rendered.

Based off `FetchKubernetesResources` though I wouldn't expect to see this resources. Is that expected or should that function be changed to render those resources as well?